src/main/java/lotto/domain/RandomNumberCreation.java

```diff
@@ -9,15 +9,15 @@ public class RandomNumberCreation implements NumberCreationStrategy {
 
     private static final int LOTTO_FIRST_NUMBER = 1;
     private static final int LOTTO_LAST_NUMBER = 45;
+    private static final List<Integer> number =
```

Comment:
`static final` 키워드를 붙여서 상수로 선언하셨으니, 컨벤션에 따라 이름을 대문자로 지어보면 어떨까요?
하나의 숫자가 아닌 로또 번호 풀을 뜻하니 단수형 보다는 복수형을 사용하는 것이 좋을 것 같네요.
```suggestion
    private static final List<Integer> LOTTO_NUMBERS =
```